model_compression_toolkit/core/keras/reader/node_builder.py

```diff
@@ -17,6 +17,9 @@
 import tensorflow as tf
 from packaging import version
 
+from mct_quantizers.logger import Logger
```

Comment:
Why get the Logger from `mct_quantizers`, and not from `model_compression_toolkit.logger`